.travis.yml

```diff
@@ -13,19 +25,23 @@ env:
 - EMBER_TRY_SCENARIO=ember-release
 - ALLOW_DEPRECATIONS=true EMBER_TRY_SCENARIO=ember-beta
 - ALLOW_DEPRECATIONS=true EMBER_TRY_SCENARIO=ember-canary
+
 matrix:
   fast_finish: true
   allow_failures:
-  - env: ALLOW_DEPRECATIONS=true EMBER_TRY_SCENARIO=ember-beta
-  - env: ALLOW_DEPRECATIONS=true EMBER_TRY_SCENARIO=ember-canary
+    - env: ALLOW_DEPRECATIONS=true EMBER_TRY_SCENARIO=ember-beta
+    - env: ALLOW_DEPRECATIONS=true EMBER_TRY_SCENARIO=ember-canary
+
 before_install:
-- npm config set spin false
-- npm install -g bower
-- npm install phantomjs-prebuilt
```

Comment:
Looks like you may have resolved this conflict in the wrong direction. Since you didn't intend to change the travis behavior, you can just revert this one file